lib/lita/handlers/cmd.rb

```diff
@@ -22,12 +22,12 @@ def run_action(resp)
         opts = resp.matches[0][1].split(" ")
 
         # the script will be the robot name if command_prefix is empty
-        return if script =~ /@?#{robot.name}/i
+        return if robot_name and script =~ /^@?#{robot_name}$/i
```

Comment:
Just thinking out loud here - maybe this line can be strengthened even further by doing:
```ruby
return if config.command_prefix.empty? and robot_name and script =~ /^@?#{robot_name}$/i
```